cumulusci/cli/cci.py

```diff
@@ -825,14 +844,19 @@ def callback(*args, **kwargs):
                 validator = import_global(validator_path)
                 validator(serv_conf)
 
-            runtime.keychain.set_service(name, ServiceConfig(serv_conf), project)
+            service_name = kwargs["service_name"]
+            runtime.keychain.set_service(
+                service_type, service_name, ServiceConfig(serv_conf), project
+            )
             if project:
-                click.echo(f"{name} is now configured for this project.")
+                click.echo(f"{service_type} is now configured for this project.")
             else:
-                click.echo(f"{name} is now configured for all CumulusCI projects.")
+                click.echo(
+                    f"{service_type} is now configured for all CumulusCI projects."
+                )
 
-        ret = click.Command(name, params=params, callback=callback)
-        return ret
+        params.append(click.Argument(["service_name"]))
+        return click.Command(service_type, params=params, callback=callback)
 
```

Comment:
github won't let me comment on the line below where the "service connect" command is registered, but let's adjust the help to say "Connect an external service to CumulusCI" instead of "Connect a CumulusCI task service"